### ISSUE TITLE
Area threshold for ironing feature

### ISSUE THREAD

**Issue by craxoor** [supermerill/SuperSlicer#3148]
Hey!

I thought it would be cool to have an area threshold for the ironing feature like for the massive infill. This would prevent the ironing of very small areas which leads to a very poor surface finish.

Currently I define a modifier cube to deactivate ironing in certain areas.

Keep up the good work!
Greetings, Greg

--------------------

Current state of ironing:
![ironing-now](https://user-images.githubusercontent.com/9419143/187154698-af5c4c12-1cac-4035-a823-da6ae0dccad5.png)

Ironing only on large surfaces:
![ironing-future](https://user-images.githubusercontent.com/9419143/187154703-1a038711-08eb-4b7e-a8ab-e370116f4e96.png)

**Comment by TerrorzwergOlli** [comment]
That would be very nice!

**Comment by supermerill** [comment]
small = small width, but can be very long ?

**Comment by craxoor** [comment]
You are right. Using an area threshold would not catch very long or wide areas. Min width/height could work better in this regard.

**Comment by ChaosBlades** [comment]
I had to use height range modifier to resolve my issue. Regardless of a feature to adjust the size of surfaces to be ironed the default is incredibly small like in my example. I would not even consider my example a top layer.
![Screenshot from 2023-05-01 06-33-59](https://user-images.githubusercontent.com/7530545/235442545-7d412738-6f16-48c9-a686-cbabbd80cd23.png)
![Screenshot from 2023-05-01 06-47-01](https://user-images.githubusercontent.com/7530545/235442546-abf3c229-9462-411b-aefb-8e829ae715e5.png)
